releases.md

```diff
@@ -63,10 +63,10 @@ The software maintainers will not back-port fixes or features to versions outsid
 
 <div class="table-styler"></div>
 
-| Major Version | Latest Minor Version |   Status    | Initial Release | Maintenance Window Start | Maintenance Window End |
-|:--------------|:---------------------|:-----------:|:---------------:|:------------------------:|:----------------------:|
-| 1             | 1.3.x                | Maintenance |  July 12, 2021  |       May 26, 2022       |   December 31, 2023    |
-| 2             | 2.0.0                |   Current   |  May 26, 2022   |           N/A            |          N/A           |
+| Major Version | Latest Minor Version |   Status    | Initial Release | Maintenance Window Start | Maintenance Window End             |
+|:--------------|:---------------------|:-----------:|:---------------:|:------------------------:|:----------------------:            |
+| 1             | 1.3.x                | Maintenance |  July 12, 2021  |       May 26, 2022       |   1 Year after 3.0 Release Date    |
```

Comment:
per the PR conversation let's change this to "GA release of 3.0." I can merge from there.

Reply:
@jkowall - wanted to make sure you saw Eli's input - thanks